contrib/bin/startKube

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+K8S_VERSION=$(curl -sS https://storage.googleapis.com/kubernetes-release/release/stable.txt)
```

Comment:
'to do this' -> to have a way to stand up a k8s cluster that is dictatorial.  There are so many ways to skin that cat.  I think the way that you have chosen makes a lot of sense and is totally reasonable for now, I'm more just thinking of the fact that this is going to be a very challenging project to test.